specs/phase0/p2p-interface.md

```diff
@@ -1298,10 +1298,10 @@ Requests are segregated by protocol ID to:
 6. Parallelise RFCs (or Eth2 EIPs).
   By decoupling requests from one another, each RFC that affects the request protocol can be deployed/tested/debated independently
   without relying on a synchronization point to version the general top-level protocol.
-  1. This has the benefit that clients can explicitly choose which RFCs to deploy
-    without buying into all other RFCs that may be included in that top-level version.
-  2. Affording this level of granularity with a top-level protocol would imply creating as many variants
-    (e.g. /protocol/43-{a,b,c,d,...}) as the cartesian product of RFCs inflight, O(n^2).
+   1. This has the benefit that clients can explicitly choose which RFCs to deploy
+      without buying into all other RFCs that may be included in that top-level version.
+   2. Affording this level of granularity with a top-level protocol would imply creating as many variants
+      (e.g. /protocol/43-{a,b,c,d,...}) as the cartesian product of RFCs inflight, O(n^2).
```

Comment:
i think the formatting change is fine although i will note that github renders the `1` and `2` sub-points as part of the outer numbered sequence so it goes `6,7,8,9` -- number `7` in the text is rendered as number `9`

likely worth fixing to get meaning across

Reply:
Hi @ralexstokes ! This is precisely what this change fixes. Currently the inner list items are actually part of the outer list in Markdown abstract syntax. With the added indentation they truly become a separate inner list and the numbering works as expected when rendered to HTML.